### ISSUE TITLE
Training on cityscapes dataset

### ISSUE THREAD

**Issue by tanveer6715** [hamidriasat/DDRNets#3]
Hi

Thanks for providing your code. I want to ask about the performance of this network on cityscapes dataset. As in the paper the author report around 77% mIoU on cityscapes dataset. But I am using your code and I am getting mIoU around 55% on validation set of cityscapes dataset. i am training it on 680X680 image resolution. Have you trained your code on cityscapes dataset?

**Comment by hamidriasat** [comment]
Hi

You're welcome.

No i haven't trained it on cityscapes dateset, it was trained on our own custom dateset and it gave us slight better results as compared to Bisenet-V2.
There can be a number of reasons you are not getting good accuracy such as
1: due to low input resolution you are getting low mIoU, increase resolution as it was described in paper, 
Remember: low resolution input always give us low accuracy.
2: pre-training on coarse cityscapes dateset increases mIoU, i think DDRNET is doing it.
3: Experiment settings  such as batch size, Optimizer, loss, initializes... also effects the training

kindly check these things and then let me know if you achieved any improvement.

**Comment by tanveer6715** [comment]
Thank you for your response. In the the paper they stated that "Cityscapes [40] is one of the well-known datasets focusing on urban street scene parsing. It contains 2975 finely annotated images for training, 500 images for validation, and 1525 images for testing. **We do not use extra 20000 coarsely labeled images during training**. There is a total of 19 classes available for the semantic segmentation task. **The resolution of images is 2048×1024, which is challenging for real-time semantic segmentation**."
It means they do not use coarse annotated images but the resolution is quite high. So that's may be the possible reason of low mIoU.

**Comment by hamidriasat** [comment]
Yes they did not used coarsely labeled images during training, but their network was pretrained on ImageNet.
check the  Train Setting section for reference.
"Before finetuning on semantic segmentation tasks, the dualresolution networks are trained on ImageNet".

And i think 55% mIou is not bad with 680 resolution, you can increase resolution and then retrain the network.

**Comment by tanveer6715** [comment]
Okay thanks for clarification. I guess you are right. I am training it from scratch with low resolution of 680. So it means there is nothing wrong in your code. Great work :)

**Comment by hamidriasat** [comment]
No issues were found.
